gen/build_deploy/powershell/dcos_install.ps1

```diff
@@ -249,11 +270,12 @@ function main($url, $version, $masters) {
     }
     $local_ip = (Get-WmiObject -Class Win32_NetworkAdapterConfiguration | where {$_.DefaultIPGateway -ne $null}).IPAddress | select-object -first 1
     Write-Log("Local IP: $($local_ip)")
-    $content = "$($masternodecontent)`n[distribution-storage]`nRootUrl=$($bootstrap_url)`nPkgRepoPath=$($version)/genconf_win/serve`nPkgListPath=latest.package_list.json`n[local]`nLocalPrivateIPAddr=$($local_ip)"
+    $content = "$($masternodecontent)`n[distribution-storage]`nRootUrl=$($bootstrap_url)`nPkgRepoPath=$($version)/genconf_win/serve/packages`nPkgListPath=$($version)/genconf_win/serve/package_lists/latest.package_list.json`n[local]`nLocalPrivateIPAddr=$($local_ip)"
     CreateWriteFile "$($basedir)\conf" "cluster.conf" $content
 
-    # Creating RunOnce scheduled task along with the RunOnce.ps1 script
-    CreateRunOnceScheduledTask $basedir $masters $local_ip
+    Write-Log("Running Winpanda.py ...")
+    & python C:\winpanda\bin\winpanda.py setup;
+    & python C:\winpanda\bin\winpanda.py start;
```

Comment:
This looks like a bug, why is winpanda in c:\?

Reply:
it's not a bug, it's a feature - winpanda unpacked into C:\
 @anatolii-borysov requested such some time back

Reply:
We had an offline discussion and for the demo I'm fine using this version. 
But after that we need to adjust to the adjustments @jongiddy requested.